save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/DemoRunComponent.kt

```diff
@@ -60,6 +58,21 @@ external interface DiktatDemoComponentProps : Props {
      * Mode for Ace Editor
      */
     var selectedMode: Languages
+
+    /**
+     * An initial value of [DemoRunRequest]
+     */
+    var emptyDemoRunRequest: DemoRunRequest
```

Comment:
Shall we pass an `emptyDemoRunRequest` with `props`? Is it done in order to not to pass `organizationName` and `projectName`?

Reply:
it's done because `emptyDemoRunRequest` is not empty for diktat for now

Reply:
Oh ok nice crutch for now